ItemPool.py

```diff
@@ -1323,6 +1323,9 @@ def get_pool_core(world):
     for item,max in item_difficulty_max[world.item_pool_value].items():
         replace_max_item(pool, item, max)
 
+    if world.damage_multiplier in ['ohko', 'quadruple'] and world.item_pool_value == 'minimal':
+        pending_junk_pool.append ('Nayrus Love')
```

Comment:
Minor formatting issue
```suggestion
        pending_junk_pool.append('Nayrus Love')
```

Reply:
Done.